test/cpu_test.cpp

```diff
@@ -2157,10 +2157,10 @@ TEST(cpu_test, tensor_copy_from_different_layout)
 
 TEST(cpu_test, max_pool_bf16)
 {
-    if (!runtime::cpu::mkldnn_utils::is_bf16_supported())
+    if (!runtime::cpu::mkldnn_utils::is_bf16_supported() || std::getenv("NGRAPH_MLIR") != nullptr)
```

Comment:
Why not just compile in the disabling of the test?

Reply:
You mean to put #ifndef outside to not compile this test when MLIR is enabled?